packages/node_modules/@webex/internal-plugin-device/test/integration/spec/device.js

```diff
@@ -962,45 +965,45 @@ describe('plugin-devices', () => {
           });
 
           afterEach('reset the request method', () => {
-            devices.request = request;
+            device.request = request;
           });
 
           it('returns a rejected promise',
-            () => assert.isRejected(devices.unregister()));
+            () => assert.isRejected(device.unregister()));
         });
       });
 
       describe('when the device is not registered', () => {
         it('should return a resolved promise',
-          () => assert.isFulfilled(devices.unregister()));
+          () => assert.isFulfilled(device.unregister()));
       });
     });
 
     describe('#waitForRegistration()', () => {
       describe('when the device is registered', () => {
-        beforeEach('register the device', () => devices.register());
+        beforeEach('register the device', () => device.register());
 
         it('should return a resolved promise',
-          () => assert.isFulfilled(devices.waitForRegistration()));
+          () => assert.isFulfilled(device.waitForRegistration()));
       });
 
       describe('when the device is not registered', () => {
         beforeEach('verify the device is not registered', () => {
-          assert.isFalse(devices.registered);
+          assert.isFalse(device.registered);
```

Comment:
Is there a way to set this manually instead of relying on an assert in a before each block?

Reply:
This can be changed, I'll investigate this further.

Reply:
Addressed these, they were redundant assertions that weren't necessary and were determined to be used for test-writing breakpoints.